### DISCUSSION TITLE
[Bug]: Can't register MariaDB as a service with `mysql_install_db`

### DISCUSSION THREAD

**Discussion by luceat-lux-vestra** [ScoopInstaller/Main#4676] [Q&A]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

mariadb

### Expected/Current Behaviour

I can't register MariaDB as a service using the following command, which is shown up after installing MariaDB.

### Steps to Reproduce

```console
scoop install mariadb
'mariadb' (10.11.2) was installed successfully!
Notes
-----
Run 'mysqld --standalone' or 'mysqld --console' to start the Database,
or run following command as administrator to register MariaDB as a service. See:
https://mariadb.com/kb/en/library/mysql_install_dbexe/

mysql_install_db --service=MariaDB --password=NewRootPassword

To stop and/or delete the Service run 'sc stop MariaDB' and 'sc delete MariaDB'.

C:\Windows\system32>mysqld --install MariaDB
Service 'MariaDB' successfully installed.

C:\Windows\system32>mysql_install_db --service=MariaDB --password=NewRootPassword
Default data directory is C:\Users\HSKIM\scoop\apps\mariadb\current\data
Running bootstrap
ERROR : Data directory C:\Users\HSKIM\scoop\apps\mariadb\current\data is not empty. Only new or empty existing directories are accepted for --datadir

C:\Windows\system32>
```
```


### Possible Solution

```
mysqld --install MariaDB
```

### Scoop and Buckets Version

```console
scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
24e91bd44 (HEAD -> master, origin/master, origin/HEAD) signal: Update to version 6.14.0

'main' bucket:
1ba361f22 (HEAD -> master, origin/master, origin/HEAD) typst: Update to version 0.2.0

'versions' bucket:
b168eb801 (HEAD -> master, origin/master, origin/HEAD) rstudio-daily: Update to version 2023.05.0-daily-217
```
```


### Scoop Config

```console
scoop config

last_update                       scoop_branch scoop_repo
-----------                       ------------ ----------
2023-04-13T14:07:07.5334894+09:00 master       https://github.com/ScoopInstaller/Scoop
```
```


### PowerShell Version

```console
$PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.22000.1335
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.22000.1335
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```
```


### Additional Softwares

_No response_

**Comment by HUMORCE** [comment]
```
> rm C:\Users\HSKIM\scoop\apps\mariadb\current\data\* -recurse
> sudo mysql_install_db --service=MariaDB --password=<pwd>
```

**Reply by luceat-lux-vestra** [reply]
TL;DR -> `mysql_install_db --service=MariaDB --password=NewRootPassword` doesn't work whether ran `mysqld --install MariaDB` first or not.

I posted with wrong sequence.
In the above example, I ran `mysqld --install MariaDB` because  
I had problem with `mysql_install_db --service=MariaDB --password=NewRootPassword`.
and `mysqld --install MariaDB` works fine.

but I wanted to note `mysql_install_db --service=MariaDB --password=NewRootPassword` is not working to this community,
and I ran `mysql_install_db --service=MariaDB --password=NewRootPassword` again.
and I posted with that sequence.
